doc/filtering.rst

```diff
@@ -92,11 +92,11 @@ This section contains functions to filter values during iteration.
 .. function:: partition(predicate, gen, param, state)
               iterator:partition(predicate)
 
-   :param x: a value to find
-   :returns: {gen1, param1, state1}, {gen2, param2, state2}
+   :returns: iterator, iterator
```

Comment:
We agreed that `iterator` is correct term for a return value and `{gen, param, state}` is incorrect one (see 'Correctness' section of my comment in the issue). But description of the `cycle()` function was not updated and still tell us about `{gen, param, state}`.